pkg/build/controller/strategy/util.go

```diff
@@ -273,6 +321,14 @@ func setupAdditionalSecrets(pod *v1.Pod, container *v1.Container, secrets []buil
 	}
 }
 
+// setupAdditionalConfigMaps creates volume mounts in the given pod for the given list of configMaps
+func setupAdditionalConfigMaps(pod *v1.Pod, container *v1.Container, configMaps []buildapi.ConfigMapSpec) {
```

Comment:
this is a mirror of the (secrets in the Custom Build Strategy)[https://docs.openshift.org/latest/dev_guide/builds/build_strategies.html#custom-secrets].  TBH I don't fully understand the use case of this feature, since secrets are supported in the `Source` part of the `BuildConfig`

Reply:
I don't see anything calling it, so let's remove both of them. (the configmap one and the secret one)

Reply:
oh sorry, i see where it's called. looking.

Reply:
I think the custom build strategy secrets field predated our generic "input secrets" support.

I think we should not introduce a configmap equivalent for custom builds, just let people use the "input configmaps" feature to inject configmap content into their custom builds.